.github/workflows/build.yml

```diff
@@ -28,15 +28,18 @@ jobs:
         include:
           - target: x86_64-unknown-linux-gnu
             os: ubuntu-latest
-            debug_features: [ rustls, pkg-config ]
+            debug_features: [ pkg-config, reflink ]
+            release_features: [ reflink ]
           - target: x86_64-apple-darwin
             os: macos-latest
+            release_features: [ reflink ]
           - target: aarch64-apple-darwin
             os: macos-latest
+            release_features: [ reflink ]
           - target: x86_64-pc-windows-msvc
             os: windows-latest
-            debug_features: [ native-tls ]
-            release_features: [ static, zlib-ng, native-tls, fancy-no-backtrace ]
+            debug_features: [ no-defaults, static, zlib-ng, native-tls, fancy-no-backtrace ]
+            release_features: [ no-defaults, static, zlib-ng, native-tls, fancy-no-backtrace, reflink ]
           - target: x86_64-unknown-linux-musl
             os: ubuntu-latest
```

Comment:
Why not enable `reflink` for the musl targets?

Reply:
they fail to build, mostly because the reflink crate is old af

Reply:
When we move to an org I'll fork reflink and get it going everywehre but for now I'm Too Tired™

Reply:
Understandable.

Let's leave a comment on #251 so that we won't forget this.